src/Compilers/CSharp/Test/Syntax/Parsing/RawStringLiteralParsingTests.cs

```diff
@@ -0,0 +1,272 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing
+{
+    public class RawStringLiteralParsingTests : ParsingTests
```

Comment:
The tests in this file don't look like parsing tests. I would extend them to verify execution as well (top-level statements + `CompileAndVerify(..., expectedOutput: ...)`)

Reply:
Gentle ping

Reply:
on it.

Reply:
ok added!